mipengine/node/tasks/views.py

```diff
@@ -64,6 +64,7 @@ def create_pathology_view(
         table_name=f"{pathology}_data",
         columns=columns,
         filters=filters,
+        privacy_protection=True,
```

Comment:
same as above, you could name the flag to something more specific as to what this flag actually does, ex. enable_min_rows_threshold

Reply:
Fixed